README.md

```diff
@@ -565,3 +565,75 @@ const results = await testSheet.testRecords(inputRows)
 expect(results).toMatchObject(expectedOutputRows)
 ```
 
+## Using git and github to collaborate with Flatfile
+
+The platform-sdk allows you to use git to manage your Flatfile config.  You can also use git to get feedback on your implementation from Flatfile engineers.  Using git can be tricky, we recommend you follow the prescribed pattern to securely share your code with Flatfile.  When you share code in this way Flatfile can much more quickly diagnose problems you are having and provide guidance.  Flatfile greatly values interaction like this because it helps us shape the future of the platform-sdk and other features.  We see your code, admire your ingenuity, and see where you're struggling then we shape the platform-sdk to ease those rough spots
+
+Steps
+- Clone the platform-sdk-starter
+```bash
+git clone git@github.com:FlatFilers/platform-sdk-starter.git
+```
+- Create your own private repo
+Navigate to https://github.com/new
+Fill out details as in the picture, take care to make the repo private and not check any other boxes.  The example repo name for this tutorial will be `private-platform-sdk-starter` and it lives at https://github.com/paddymul/private-platform-sdk-starter (check that your new repo isn't accessible from a non-logged in private browsing tab before proceeding)
+
+![Create a private repo](/assets/GH-create-private-repo.png)
+- Push to your new private repo
+``` 
```

Comment:
⛳ I think you could add `bash` here even with the comments below and it wont complain